plugin/discovery/find.go

```diff
@@ -96,6 +96,7 @@ func findPluginPaths(kind string, machineName string, dirs []string) []string {
 				continue
 			}
 
+			// FIXME: we pass in GOOS_GOARCH paths directly, so these may not be "legacy"
```

Comment:
Ahh, okay... so this `FIXME` is actually for the callers, not for this function itself. Makes sense now, though I expect we will just lose track of this since it's not close to the thing that needs fixed. :grinning: